javascript/builtins/String.json

```diff
@@ -158,6 +158,58 @@
             }
           }
         },
+        "at": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String/at",
+            "spec_url": "https://tc39.es/proposal-relative-indexing-method/#sec-string-prototype-additions",
+            "support": {
+              "chrome": {
+                "version_added": "89"
+              },
+              "chrome_android": {
+                "version_added": "89"
+              },
+              "edge": {
+                "version_added": "89"
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": "89"
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": true
```

Comment:
```suggestion
              "deprecated": false
```

Reply:
🤦🏻‍♀️ I'm so glad you're reviewing @Elchi3 - I feel I've learnt a lot here today ☺️